client/a8c-for-agencies/sections/marketplace/products-overview-v2/product-action-panel/bundle-price-selector.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button } from '@wordpress/components';
+import { Icon, chevronDown } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useRef, useState } from 'react';
+import PopoverMenu from 'calypso/components/popover-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+
+type Props = {
+	options: number[];
+	value: number;
+	onChange: ( value: number ) => void;
+};
+
+const getDiscountPercentage = ( bundleSize: number ) => {
+	// FIXME: Need to calculate based on average discount per bundle size.
+	return (
+		{
+			1: '',
+			5: '10%',
```

Comment:
Should these percentages be localized. I think there are locales that put the percent sign before the number – I'm not 100% sure on that though.

Reply:
Good point. I've made it localized. 👍